hw/top_earlgrey/util/vivado_hook_write_bitstream_pre.tcl

```diff
@@ -13,3 +13,49 @@ send_msg "Designcheck 1-2" INFO "Slack is ${slack_ns} ns."
 if [expr {$slack_ns < 0}] {
   send_msg "Designcheck 1-3" ERROR "Timing failed. Slack is ${slack_ns} ns."
 }
+
+set filename "rom.mmi"
+set fileout [open $filename "w"]
+set brams [split [get_cells -hierarchical -filter { PRIMITIVE_TYPE =~ BMEM.bram.* && NAME =~ *u_rom_rom*}] " "]
+
+# Should merge the two loops later
+# calculate the overall address space
+set space 0
+foreach inst [lsort -dictionary $brams] {
+    set slice_begin [get_property bram_slice_begin [get_cells $inst]]
```

Comment:
nit: indent with two spaces for consistency with existing code in this file.

Reply:
Done